### ISSUE TITLE
Find a home base location for Sunday

### ISSUE THREAD

**Issue by reidab** [wherecamppdx/planning#9]

**Comment by geografa** [comment]
possible locations:
- Esri
- Urban Airship
- Olympic Mills
- Ristretto Roasters (SE Couch or N Williams)
- Coava

**Comment by ungoldman** [comment]
September 14th right? Pinged @aaronpk, I think @EsriPDX might be available to host

**Comment by geografa** [comment]
yes. :+1:  - just let me know by end of next week and we'll get it on the site.

**Comment by aaronpk** [comment]
Just realized this is the same weekend as XOXO. I won't be able to host at EsriPDX, but it's open if someone else from our office wants to!
